tests/menubar-editor/navigating-to-menubar-interaction.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>

```

Comment:
If these files are being generated from a csv, it might be better to ignore these files in git and generate them in the build process. That would help to reinforce the csv as the authoritative source and prevent someone from modifying the generated files.

Reply:
I agree this is fine for now, but we should revisit the whole test creation workflow. Hopefully after we get this first set of tests are in we can do some strategizing here (informed by this first set of test writing), and Bocoup can spend some time on it in the next month.

We have one tricky area -- right now, most of the data is declarative, but there is the optional `setupTestPage` script to get the widget in the correct state before the test. I don't think we want to write javascript in a spreadsheet or CSV file. It's possible we could use the .html test files as an input, along with the csv files, as well as an output, for a test generation script. For example: If an html file exist with that name, we can rewrite all the declarative pieces of information from a CVS file, and keep the `setupTestPage` from the test file itsself. Otherwise, create a new html page.